raster/r.watershed/ram/init_vars.c

```diff
@@ -83,7 +83,7 @@ int init_vars(int argc, char *argv[])
         else if (sscanf(argv[r], "blocking=%s", ob_name) == 1)
             ob_flag++;
         else if (sscanf(argv[r], "disturbed_land=%s", ril_name) == 1) {
-            if (sscanf(ril_name, "%lf", &ril_value) == 0) {
+            if (sscanf(ril_name, "%lf", &ril_value) == 0 || sscanf(ril_name, "%lf", &ril_value) == EOF ){
```

Comment:
`ril_name` was successfully read from the previous `sscanf()`, so it shouldn't be `"\0"` and there is no need for this `EOF` check.

Reply:
i removed this change.